examples/deployment/kubernetes/trillian-log-signer-deployment.yaml

```diff
@@ -18,30 +18,30 @@ spec:
         secret:
           secretName: trillian-key
       nodeSelector:
-        cloud.google.com/gke-nodepool: signer-pool
+        cloud.google.com/gke-nodepool: logsigner-pool
```

Comment:
Would be nice if it was consistently either "log-signer" or "logsigner".

Reply:
Agreed. Will fix.